mappings/net/minecraft/entity/player/EntityPlayer.mapping

```diff
@@ -141,6 +141,7 @@ CLASS none/aam net/minecraft/entity/player/EntityPlayer
 	METHOD cS getSleepTimer ()I
 	METHOD cT getSpawnPosition ()Lnone/cn;
 	METHOD cU isSpawnForced ()Z
+	METHOD cV getXpSeed ()I
```

Comment:
It's used to seed the rng that decides what enchantments are available at the enchantment table.